__tests__/app.test.js

```diff
@@ -285,3 +285,71 @@ describe("CORE: POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
```

Comment:
I would like to see another test added here for 
Status 400, incorrect body, e.g. inc_votes property is not a number

But your logic is spot on !